### DISCUSSION TITLE
Support for React 18

### DISCUSSION THREAD

**Discussion by jan-bock** [rjsf-team/react-jsonschema-form#3871] [General]
Is there an estimated release date for React 18 support?
Currently undergoing a dependency update including a React upgrade to v.18 so keen to understand the current state of the work :)

**Comment by yassine-bhn** [comment]
Modern versions of the library officially support React 18.
In fact, newer releases dropped support for React versions below 18, meaning React 18 is now the baseline

**Reply by bargaouiahmed** [reply]
thank you
